src/containers/flows/BuyNow/index.tsx

```diff
@@ -143,11 +144,11 @@ const BuyNowContainer = ({
 			console.warn('<BuyNow> Failed to Get Data', e);
 		}
 		try {
-			const [domain, metadata, balance] = await Promise.all([
+			const [domain, balance] = await Promise.all([
 				sdk.getDomainById(domainId),
-				sdk.getDomainMetadata(domainId, library.getSigner()),
 				wildContract.balanceOf(account),
 			]);
+			const metadata = await getMetadata(domain.metadataUri);
 			if (domain && metadata && buyNowPrice && balance) {
```

Comment:
@colbr  Can we use `sdk.utility.getMetadataFromUri(uri)` instead of using `getMetadata(uri)`. As we wanted to get rid of all the direct interactions.

Reply:
Unfortunately metadata has to be parsed to process default values properly. For example, the SDK will return `isBiddable: undefined`, which means wherever we use it we have to do `isBiddable ?? true` since `isBiddable` should default to `true` if it's `undefined`.

I'll make some cleaner code for this, let me do another push

Reply:
Yeah makes sense. Probably a transform method. Sure, I'll wait for your changes.